### DISCUSSION TITLE
Getiing error "error": { "clientVersion": "3.13.0" }

### DISCUSSION THREAD

**Discussion by Jhaamlal** [prisma/prisma#13177] [Q&A]
I am getting this error even after deleting node module and restalling it.

his is repository - https://github.com/Jhaamlal/MemebershipBackend/

File in which error = `src/authentications/customerAuth.ts`
line to 11-13

while sending post request at Line 14 in INDEX.ts `/Customer`

dummy data you can use to replicate :
```
{ "email":"shank@gmail.com", "firstName":"ravi", "lastName":"singh", "user":{ "email":"shank@gmail.com", "password":"12345678", "phone":"9166186442" }, "address":{ "address_line":"p-124/2 ghotic line", "city":"jaipur", "state":"Rajsthan", "PIN":8002204 } }
```

**Comment by nurul3101** [comment]
Hey @Jhaamlal 👋 
Did you only get the clientVersion in error response?
Did you get any error code? It seems your repository is private.

**Reply by Jhaamlal** [reply]
Yes @nurul3101  I have solved that issue. Actually issue is that bro ,Prisma is very bad at telling issue exactly it's gives very general error. Please improve that part.

**Reply by kxzo** [reply]
How did you fix it, @Jhaamlal ? I have the same problem :c

**Reply by pantharshit00** [reply]
Hello @einKnuffy,

Were you able to solve your problem? Can you please report it in a new discussion if it is not addressed.

**Reply by kxzo** [reply]
Somehow it stopped occuring...

**Reply by vladi-stevanovic** [reply]
Hi @einKnuffy thank you for following up. We're glad to hear that this is not happening anymore and in case you come across it again please let us know the details, so we can try to reproduce it on our end! 

@Jhaamlal thank you for confirming it was solved on your end too! If you have a moment, please share the solution that you found so that other users may try it if they come across the same issue. It would be very helpful to have it documented!

**Reply by leopedroso45** [reply]
So? How did you solve the problem?????

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
